Solutions/Threat Intelligence/Analytic Rules/IPEntity_imWebSession.yaml

```diff
@@ -83,8 +72,7 @@ customDetails:
   IoCConfidenceScore: ConfidenceScore
 
 alertDetailsOverride:
-  alertDisplayNameFormat: The IP {{SrcIpAddr}} of a web request to hostname  {{domain}} matched an IoC
-  alertDescriptionFormat: The source address {{SrcIpAddr}} of a web request for URL {{Url}} matched a known indicator of compromise of {{ThreatType}}. Consult the threat intelligence blead for more information on the indicator.
-
-version: 1.1.1
+  alertDisplayNameFormat: "The IP {{SrcIpAddr}} of a web request to hostname {{domain}} matched an IoC"
```

Comment:
Values need not be in quotes.
Plus value of "domain" is not present in the schema of the query.

Reply:
Fixed the issue, Replaced the domain with src ip,